packages/crew_url_opener.rb

```diff
@@ -0,0 +1,104 @@
+require 'package'
+
+class Crew_url_handler < Package
+  description 'Script for opening an URL in Chrome OS browser'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '1.0'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'SKIP'
+
+  @_www_browser_symlinks = ["#{CREW_PREFIX}/bin/x-www-browser", "#{CREW_PREFIX}/bin/gnome-www-browser"]
+
+  def self.preflight
+    # check if `UrlHandlerService` service exist
+    @_service = Kernel.system <<~CMD
+      dbus-send --system \
+        --dest=org.chromium.UrlHandlerService \
+        --type=method_call \
+        --print-reply \
+        /org/chromium/UrlHandlerService \
+        org.freedesktop.DBus.Peer.Ping
+    CMD
+
+    unless @_service
+      abort "This package is not compatible with your device :/".lightred
+    end
+  end
+
+  def self.build
+    @_script = <<~SCRIPT
+      #!/bin/bash
+      case "${1}" in
+      ''|-h|--help)
+        cat <<EOF
+      ${0##*/}: Open an URL in Chrome OS browser.
```

Comment:
Should this say `Open an URL in your default browser`?

Reply:
But users may select a different browser as the default